README.md

```diff
@@ -1,2 +1,186 @@
 # react-math-magicians
-Math magicians is a website for all fans of mathematics. It is a Single Page App (SPA) built with React using Routers, It allows users to make simple calculations and read a random math-related quote.
+
+<div align="center">
+  <img  src="./src/assets/images/react-math-magicians.png" alt="logo" width="1080"  height="auto" />
+  <br/>
+
+  <h3><b>react-math-magicians</b></h3>
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [Walkthrough Video](#walkthrough-video)
+  - [🛠 Built With](#built-with)
+  - [Tech Stack](#tech-stack)
+  - [Key Features](#key-features)
+- [🧑🏻‍💻 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Clone](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+# 📖 React Math Magicians <a name="about-project"></a>
+
+> React Math Magicians is a website for all fans of mathematics. It is a Single Page App (SPA) built with React using Routers, It allows users to make simple calculations and read a random math-related quote.
+
+
+## Walkthrough Video <a  name="walkthrough-video"></a>
+Comming Soon
+
+## 🛠 Built With <a name="built-with"></a>
+
+- Visual Studio Code
+- HTML
+- CSS
+- Javascript
+- Github
+- React
+- Routes
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://en.wikipedia.org/wiki/HTML">Html</a></li>
+    <li><a href="https://en.wikipedia.org/wiki/CSS">Css</a></li>
+    <li><a href="https://en.wikipedia.org/wiki/CSS">Javascript</a></li>
+    <li><a href="https://es.wikipedia.org/wiki/React">React</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+#### Key Features <a name="key-features"></a>
+
+ <ul>
+  <li>HTML Interface with CSS (Desktop / mobile)</li>
+  <li>JavaScript</li>
+  <li>React</li>
+
+ </ul>
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🧑🏻‍💻 Live Demo <a name="live-demo"></a>
+
+Comming Soon
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and use as a template for your own portafolio site, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need: git installed in your system
+
+### Clone
+
+Clone this repository to your desired folder:
+
+```sh
+  git clone https://github.com/adelphinsucasa/react-math-magicians.git
+```
+
+### Install
+
+Install this project with:
+
+```sh
+  npm init -y
+  npm install
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+  npm start
+```
```

Comment:
- [ ] This command does not seem to run the app. Kindly put in the right command to run the project. 🤝️